### PR TITLE
Added second Algorithm for BubbleSort

### PR DESCRIPTION
**Pull request by AndraB90**
A variant to first BubbleSort Algorithm

### CODE REVIEW

File: src/bubble_sort/Example2.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package bubble_sort;
+
+import java.util.Random;
+
+public class Example2 {
+    public static void main(String[] args) {
+        int[] array = new int[10];
+        Random generator = new Random();
+        for (int i = 0; i < array.length; i++) {
```
Comment (CorlyBoy) [inline src/bubble_sort/Example2.java:9]: please leave a empty line before basic procedures (for, while, if, etc.)

File: src/bubble_sort/Example2.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            array[i] = generator.nextInt(10001);
+        }
+        System.out.println("Before sorting: ");
+        printArray(array);
+
+        boolean isSorted;
+
+        for (int i = 0; i < array.length; i++) {
+            isSorted = true;
+            for (int j = 1; j < array.length-i; j++) {
```
Comment (CorlyBoy) [inline src/bubble_sort/Example2.java:19]: same as above

File: src/bubble_sort/Example2.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                    swapValues(array, j,j-1);
+                    isSorted = false;
+                }
+            }
+            if(isSorted){
+                break;
+            }
+        }
+
+
```
Comment (CorlyBoy) [inline src/bubble_sort/Example2.java:30]: too many empty lines

File: src/bubble_sort/Example2.java (hunk 1 above)
Comment (AndraB90) [reply src/bubble_sort/Example2.java:9]: Done

File: src/bubble_sort/Example2.java (hunk 2 above)
Comment (AndraB90) [reply src/bubble_sort/Example2.java:19]: Done

File: src/bubble_sort/Example2.java (hunk 3 above)
Comment (AndraB90) [reply src/bubble_sort/Example2.java:30]: Resolved

**Review by CorlyBoy — APPROVED** [review]
Looks good!

**Pull request merged**